tests/test_smartredis.py

```diff
@@ -31,7 +31,7 @@
 )
```

Comment:
At the beginning of this file, there is a `REDIS_PORT` constant which we can now delete (and use your new utility wherever it appeared).